### DISCUSSION TITLE
Themed Favicons support (and a New Epic Stack Logo!!)

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#201] [Announcements]
![Logo with four identical rocket shapes pointing out and the words "Epic Stack"](https://github.com/epicweb-dev/epic-stack/assets/1500684/1b00286c-aa3d-44b2-9ef2-04f694eb3592)

You can find branding assets at [EpicWeb.dev/brand](https://www.epicweb.dev/brand).

I've also update the favicon for the app. Naturally folks will want to use their own favicon, but it's a cool one to start off with and it also demonstrates how to get a themed icon as well. In fact, I've recorded a video demonstrating how it works!

[![Video of Kent sharing his screen and giving a peace sign](https://github.com/epicweb-dev/epic-stack/assets/1500684/0c4b53d5-29ed-49c6-b371-19e127e5c80f)](https://www.epicweb.dev/tips/responsive-favicons)

Watch [Create a Responsive Favicon for Light and Dark Mode with CSS](https://www.epicweb.dev/tips/responsive-favicons)

**Comment by kentcdodds** [comment]
Stay tuned, found an even better way to do this. I'll have to re-record this video 😅

https://twitter.com/andreisena/status/1670866984848719892

**Reply by kentcdodds** [reply]
Done 👍

**Comment by carlosvirreira** [comment]
Love the new branding!
